cypress/integration/createBoxesAndProducts.js

```diff
@@ -0,0 +1,106 @@
+import uuidv4 from "uuid/v4";
+
+describe('Create and delete boxes', function() {
+    let testUserMail;
+    let testPwd;
+
+    before(function() {
+        cy.getTestData().then(($result) => {
+            testUserMail = $result.testUserMail;
+            testPwd = $result.testPwd;
+        });
+    })
+
+    beforeEach(function() {        
+        cy.reLogin(testUserMail, testPwd);
+    });
+
+    afterEach(function() {
+        cy.wait(2000);
+    });
+
+    it('Create new product', () => {
+        let productName = uuidv4().substring(0,6);
+        cy.navigateToProductsPage();
+        cy.get('button[data-cy=addProductButton]').click();
+        cy.get('#select-category').click();
```

Comment:
I think there's a data-cy added for this?

Reply:
Yupp, my bad I've forgotten :) Fixed.